src/backend/core/authentication/backends.py

```diff
@@ -220,7 +219,7 @@ def compute_full_name(self, user_info):
         """Compute user's full name based on OIDC fields in settings."""
         name_fields = settings.USER_OIDC_FIELDS_TO_NAME
         full_name = " ".join(
-            user_info[field] for field in name_fields if user_info.get(field)
+            user_info.get(field) for field in name_fields if user_info.get(field)
```

Comment:
This change does nothing because `user_info[field]` is always ok when `user_info.get(field)` is truthy.